wcc-contentful/lib/wcc/contentful/middleware/store/caching_middleware.rb

```diff
@@ -19,9 +17,9 @@ def find(key, **options)
           event = 'miss'
           # if it's not a contentful ID don't hit the API.
           # Store a nil object if we can't find the object on the CDN.
-          (@cdn.find(key, options) || nil_obj(key)) if key =~ /^\w+$/
+          (store.find(key, options) || nil_obj(key)) if key =~ /^\w+$/
         end
-      _instrument(event + '.lazycachestore', key: key, options: options)
```

Comment:
is this event context still included somewhere?

Reply:
I renamed it.  Two things I did with store instrumentation:

1. `find.store.contentful.wcc`, `find_by`, etc are being sent by the `InstrumentationMiddleware` which is by default at the top of the middleware stack.  They are no longer being sent by individual stores.
2. The lazy-cache specific instrumentation is using the class name now as it's namespace.  I changed the class name b/c it's a middleware now.